code/modules/mob/living/simple_animal/borer/borer_html.dm

```diff
@@ -0,0 +1,129 @@
+
+/mob/living/simple_animal/borer/proc/get_html_template(content)
+	var/html = {"<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" \"http://www.w3.org/TR/html4/loose.dtd\">
+		<html>
```

Comment:
Probably because tgui is fairly new, most servers afaik don't use it, and like 3 people know how to write it. I know refactoring things to match our interfaces is in the scope of a port, but I would give him a break here. This can definitely be fixed in post, and using it as an excuse to hold up the whole feature is practically like begging him give up on the PR.